openapi-cli/src/main/java/io/ballerina/openapi/generators/client/FunctionSignatureGenerator.java

```diff
@@ -388,26 +388,30 @@ private Node getHeaderParameter(Parameter parameter, NodeList<AnnotationNode> pa
                     false));
             if (schema.getDefault() != null) {
                 BuiltinSimpleNameReferenceNode typeName = createBuiltinSimpleNameReferenceNode(null,
-                        createIdentifierToken(convertOpenAPITypeToBallerina(
-                                parameter.getSchema().getType().trim())));
+                        createIdentifierToken(convertOpenAPITypeToBallerina(parameter.getSchema().getType().trim())));
                 LiteralValueToken literalValueToken;
                 if (schema.getType().equals("string")) {
                     literalValueToken = createLiteralValueToken(null,
                             '"' + schema.getDefault().toString() + '"', createEmptyMinutiaeList(),
                             createEmptyMinutiaeList());
                 } else {
-                    literalValueToken =
-                            createLiteralValueToken(null, schema.getDefault().toString(),
-                                    createEmptyMinutiaeList(),
-                                    createEmptyMinutiaeList());
-
+                    literalValueToken = createLiteralValueToken(null, schema.getDefault().toString(),
+                                    createEmptyMinutiaeList(), createEmptyMinutiaeList());
                 }
                 return createDefaultableParameterNode(parameterAnnotationNodeList, typeName, paramName,
                         createToken(EQUAL_TOKEN), literalValueToken);
             } else {
+                String type = convertOpenAPITypeToBallerina(parameter.getSchema().getType().trim()) + "?";
```

Comment:
Don't we have a constant for `"?"`

Reply:
Used the constant in https://github.com/ballerina-platform/ballerina-openapi/commit/42d8cb145f00ec8152ad6c2d6d3dd9b1d2909dcb